src/Symfony/Component/Validator/ConstraintValidator.php

```diff
@@ -87,8 +87,14 @@ protected function formatValue($value, $format = 0)
     {
         if (($format & self::PRETTY_DATE) && $value instanceof \DateTimeInterface) {
             if (class_exists('IntlDateFormatter')) {
+                $timezone = $value->getTimezone();
+                // Timezone 'Z' is a valid ISO8601 indicator for UTC
+                $timezone = 'Z' === mb_strtoupper($timezone->getName()) ? new \DateTimeZone('UTC') : $timezone;
```

Comment:
Do we really need to use `mb_strtoupper` here?

Reply:
@fabpot I've removed `mb_` prefix